src/responder/kcm/kcmsrv_cmd.c

```diff
@@ -311,8 +311,12 @@ static void kcm_reply_error(struct cli_ctx *cctx,
 {
     errno_t ret;
     krb5_error_code kerr;
+    int level = SSSDBG_OP_FAILURE;
 
-    DEBUG(SSSDBG_OP_FAILURE,
+    if (retcode == ERR_KCM_OP_NOT_IMPLEMENTED) {
+        level = SSSDBG_MINOR_FAILURE;
+    }
+    DEBUG(level,
```

Comment:
Hi,

since you use it only once wouldn't `retcode == ERR_KCM_OP_NOT_IMPLEMENTED ? SSSDBG_MINOR_FAILURE : SSSDBG_OP_FAILURE` have a similar readability with less code?

bye,
Sumit

Reply:
Ok, done.